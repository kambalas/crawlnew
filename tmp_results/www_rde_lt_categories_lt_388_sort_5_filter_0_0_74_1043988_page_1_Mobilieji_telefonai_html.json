[
  {
    "name": "iPhone 14 Plus 128GB (PRODUCT)RED Apple",
    "price": "736.43",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB Yellow Apple",
    "price": "805.35",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 256GB (PRODUCT)RED Apple",
    "price": "852.66",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB Midnight Apple",
    "price": "918.39",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB Blue Apple",
    "price": "918.39",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 256GB Midnight Apple",
    "price": "1018.64",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 256GB Purple Apple",
    "price": "1051.18",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB Purple Apple",
    "price": "1086.00",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 512GB Midnight Apple",
    "price": "1359.00",
    "error": false
  }
]