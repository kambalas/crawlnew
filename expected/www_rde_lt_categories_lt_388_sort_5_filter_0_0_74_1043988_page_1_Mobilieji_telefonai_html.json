[
  {
    "name": "iPhone 15 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 15 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 16 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone SE 4",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone SE",
    "price": "",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB (PRODUCT)RED Apple",
    "price": "736.43",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB Yellow Apple",
    "price": "805.35",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB Midnight Apple",
    "price": "918.39",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB Blue Apple",
    "price": "918.39",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 128GB Purple Apple",
    "price": "1051.18",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 256GB (PRODUCT)RED Apple",
    "price": "852.66",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 256GB Midnight Apple",
    "price": "1014.65",
    "error": false
  },
  {
    "name": "iPhone 14 Plus 256GB Purple Apple",
    "price": "1086.00",
    "error": false
  },
  {
    "index": 3,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  },
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'list' object has no attribute 'usage'"
  }
]